ckanext/depositar_theme/fanstatic/scss/_index.scss

```
@@ -0,0 +1,15 @@
+@import 'index/variables';
+@import 'index/banner';
+@import 'index/intro';
+@import 'index/features';
+@import 'index/showcases';
+@import 'index/stats';
+@import 'index/guide';
+
+// @import 'topics';
```

```suggestion
@import 'index/topics';
```
We still import styles for the `topics` page just in case we re-enable it.